lib/attempt.go

```diff
@@ -69,6 +73,19 @@ type Task struct {
 	IsGroup      bool          `json:"isGroup"`
 }
 
+type logfiles struct {
```

Comment:
`lib/attempt.go` should related attempt. (e.g. spath contains `/attempts`)
So if you want to these changes, could you create a new file (`lib/log.go`) and put it in there?